tools/ci/templates/jobs-libwebrtc-win32.yaml

```diff
@@ -44,19 +54,34 @@ jobs:
       configuration: ${{parameters.buildConfig}}
     timeoutInMinutes: 120
 
-  # Publish webrtc.lib
+  # Publish webrtc.lib as pipeline artifacts (limited retention)
   - task: PublishPipelineArtifact@0
     displayName: 'Publish webrtc.lib ($(buildTriple))'
     inputs:
       artifactName: 'libwebrtc_$(buildTriple)'
       targetPath: 'external/webrtc-uwp-sdk/webrtc/xplatform/webrtc/OUTPUT/webrtc/$(scriptPlatform)/${{parameters.buildArch}}/${{parameters.buildConfig}}/webrtc.lib'
     timeoutInMinutes: 15
 
-  # Publish PDBs for webrtc.lib
-  - task: PublishPipelineArtifact@0
+  # Publish PDBs for webrtc.lib as pipeline artifacts (limited retention)
+  # - task: PublishPipelineArtifact@0
+  #   displayName: 'Publish PDBs for webrtc.lib ($(buildTriple))'
+  #   inputs:
+  #     artifactName: 'libwebrtc_pdbs_$(buildTriple)'
+  #     targetPath: 'external/webrtc-uwp-sdk/webrtc/xplatform/webrtc/OUTPUT/webrtc/$(scriptPlatform)/${{parameters.buildArch}}/${{parameters.buildConfig}}/pdbs'
+  #   timeoutInMinutes: 15
+
+  # Publish PDBs for webrtc.lib as Universal Package (unlimited retention)

```

Comment:
Why do we need to keep the pdbs indefinitely (especially given that the lib is only kept as a pipeline artifact)?

Reply:
There are 2 distinct data streams here for `webrtc.lib` and its PDBs:
- If we are considering only CI, then we need the `webrtc.lib` PDBs for the C++ CI. The former runs infrequently, and pipeline artifacts have a limited retention period, so it's not uncommon for all artifacts on the core build to be deleted, which leaves the C++ CI without anything. Therefore we use Universal Packages instead which have unlimited retention (like a NuGet package, but without size limit).
- For `webrtc.lib` itself we are only publishing it here as a pipeline artifact because we need to move it from that pipeline to the signing one, which will generate the NuGet package we publish on nuget.org. This generally happens within a few hours or less. After that, the C++ CI will use the published NuGet packages, and the pipeline artifacts can be deleted.

Reply:
Hmm I see. In general anyone who wants to build the MR.WebRTC.Native dlls will need/want the pdbs to be around, I suppose.